egov-persister/property-services-registry.yml

```diff
@@ -19,7 +25,7 @@ serviceMaps:
         dbType: JSONB
 
     - query: INSERT INTO eg_pt_property (id, propertyid, tenantid, surveyid, accountid, oldpropertyid, status, acknowldgementnumber, propertytype, ownershipcategory, usagecategory, creationreason, nooffloors, landarea, superbuiltuparea, linkedproperties, source,  channel, createdby, createdtime, lastmodifiedby, lastmodifiedtime, additionaldetails) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?);
-      basePath: Property
+      basePath: $.Property
```

Comment:
Why is this change needed?

Reply:
shashwat asked me to make this change by referring to this branch: https://github.com/egovernments/configs/blob/core-113/egov-persister/property-services-registry.yml
This branch is not yet merged, so  I think he asked me to replicate these changes so the code won't break.